servers.json

```diff
@@ -1653,9 +1653,14 @@
         "id": "nigxht",
         "name": "Nigxht Network",
         "addresses": [
-            "nigxht.net",
-	    "mc.nigxht.us",
-	    "play.nigxht.us"
+            "nigxht.net"
```

Comment:
You're allowed to have your other .us domain, we just don't want sub-domains.

Reply:
Now only have nigxht.net